src/resmom/mom_main.c

```diff
@@ -10685,36 +10684,48 @@ check_busy(double mla)
  *		contain a different type of information, this function will need
  *		to change.
  *
+ *		On Windows we use native Windows API's to discover the topology
+ *
  * @see	dep_topology()
  *
  */
 void
 mom_topology(void)
 {
 #ifndef NAS /* localmod 113 */
-	extern char		mom_short_name[];
-	extern callfunc_t	vn_callback;
-	int			ret;
-	hwloc_topology_t	topology;
-	char			*xmlbuf;
-	int			xmllen;
-	vnl_t			*vtp = NULL;
+	extern char mom_short_name[];
```

Comment:
Since the changes you're making are within a NAS section, have you tried building with -DNAS? We generally avoid modifying NAS specific sections.

Reply:
I tried compiling in NAS mode. But it failed in multiple other places. And I don't think this code will add anything to that as it is under `#ifndef NAS`